conf/routes

```diff
@@ -352,15 +352,14 @@ POST  /streamer/:username/check        controllers.Streamer.checkOnline(username
 
 # Private Play
 
-GET   /local                           controllers.Local.index
-GET   /local/bots                      controllers.Local.bots
```

Comment:
we could leave the xhr bot list route in place until your new ui makes it unnecessary. with current code, it's still necessary